spec/locale.html

```diff
@@ -350,4 +350,35 @@ <h1>get Intl.Locale.prototype.region</h1>
       </emu-alg>
     </emu-clause>
   </emu-clause>
+
+  <emu-clause id="sec-properties-of-intl-locale-instances">
+    <h1>Properties of Intl.Locale Instances</h1>
+
+    <p>
+      Intl.Locale instances are ordinary objects that inherit properties from %Locale.prototype%.
+    </p>
+
+    <p>
+      Intl.Locale instances have an [[InitializedLocale]] internal slot.
+    </p>
+
+    <p>
+      Intl.Locale instances also have several internal slots that are computed by the constructor:
+    </p>
+
+    <ul>
+      <li>[[Locale]] is a String value with the language tag of the locale whose localization is used for formatting.</li>
+      <li>[[Calendar]] is a String value with the *"type"* given in Unicode Technical Standard #35 for the calendar used for formatting.</li>
+      <li>[[Collation]] is a String value with the *"type"* given in Unicode Technical Standard #35 for the collation, except that the values *"standard"* and *"search"* are not allowed, while the value *"default"* is allowed.</li>
```

Comment:
I may be either overthinking or underthinking this one. Reasoning:  In Intl.Collator, "search" and "standard" aren't allowed as options for "co" and aren't allowed as values for [[Collation]]. They were banned in Intl.Collator in order to force people to use the `usage` option. Implementations of Intl.Collator appear to respect that constraint. See https://github.com/tc39/ecma402/issues/256#issuecomment-411421731 for a discussion of this.

Since the value of Intl.Collator's [[Collation]] slot can't be "search" or "standard", does it make sense for the analogous slot in Intl.Locale to allow them? I admit that the answer here might just be "well of *course* it should allow those values." 

One argument for not including that constraint for the present moment might be that if the constraint is imposed,  the correction of an editorial oversight would result in a normative change.

Reply:
Upon further investigation: The MDN entry for Intl.Locale.prototype.getCollations() currently has a warning against use of "search" that mentions the constraint in Intl.Collator.  A warning also exists for "standard", but because of collation naming inconsistencies in Swedish, which I presume is what motivated the ban in "standard" in Intl.Collator.

Reply:
Presumably yes. But let's keep this PR editorial and open an issue for whether or not Intl.Locale should align with Intl.Collator on this point (for which I can imagine arguments in favor of either resolution).